ipatests/test_integration/test_installation.py

```diff
@@ -227,6 +227,23 @@ def test_replica2_ipa_kra_install(self):
         super(TestInstallWithCA2, self).test_replica2_ipa_kra_install()
 
 
+class TestInstallCA(IntegrationTest):
+
```

Comment:
Can you add comments to the code? For instance for the class
```
""" Tests for CA installation on a replica
"""
```
and for each test method
```
"""
Test for ipa-ca-install --no-host-dns on a replica
"""
```
and
```
"""
Test for ipa-ca-install --skip-schema-check on a replica
"""
```